content/contribute.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contributing to gRPC
+---
+
+To contribute to gRPC documentation, please fork the [gRPC repository on GitHub](https://github.com/grpc/grpc.io) and start submitting pull requests.
+
+## Contribute guidelines for gRPC
+
+We welcome contributions to any of our three core repositories:
+
+* [gRPC](https://github.com/grpc/grpc)
+* [gRPC Java](https://github.com/grpc/grpc-java)
+* [gRPC Go](https://github.com/grpc/grpc-go)
+
```

Comment:
Seems a bit strange to have a contributing doc over grpc.io that only mentions contributions to "three core" repos that don't include grpc.io

Reply:
Again, I haven’t modified any content, as it’s out of scope for this PR

Reply:
That works for me.